### DISCUSSION TITLE
Radiomaster Boxer Companion 2.8.1

### DISCUSSION THREAD

**Discussion by coaster007** [EdgeTX/edgetx#3330] [Q&A] [answered]
the  boxer radio has the option to move models, but in the companion i haven't found this option.
how to move models in companion ?

**Comment by pfeerick** [comment, marked as answer]
You mean change the order of models? You click and drag (while holding the SHIFT key on the keyboard down if you want to move instead of copy - the little `+` next the mouse cursor will change to an arrow). Also, make sure you are releasing the model between two others, not on top of an existing one, otherwise that will prompt for overwrite/insert.

![companion_EOdH0TbyB7](https://user-images.githubusercontent.com/5500713/223877079-2f781e08-4fd1-4311-a320-402411f06982.gif)

**Reply by coaster007** [reply]
is that simple !
until now i changed the order of models like the game "towers of hanoi"  :-)

thank's a lot for the quick answer !!!

greetings from italy
Graziano
